models.py

```diff
@@ -5,34 +5,54 @@
 db = SQLAlchemy(app)
 db.reflect()
 
+
 class HadithCollection(db.Model):
     __tablename__ = 'Collections'
 
     def serialize(self):
         return {
             'name': self.name,
-            'short_intro': self.shortintro,
             'has_books': self.hasbooks == "yes",
             'has_chapters': self.haschapters == "yes",
-            'title_en': self.englishTitle,
-            'title_ar': self.arabicTitle,
+            'collection': [
+                {
+                    'lang': 'en',
+                    'title': self.englishTitle,
+                    'short_intro': self.shortintro,
+                },
+                {
+                    'lang': 'ar',
+                    'title': self.arabicTitle,
+                    'short_intro': self.shortintro,
```

Comment:
Can we make this conditional if the field `shortIntroArabic` exists and None or empty string otherwise? And the field name change to `shortIntroArabic`.

Reply:
arabic is already in the language code, why do we want to add it as suffix?

Reply:
I mean the `self.shortintro` field - it is English text only in the database, and if I were to add an Arabic short intro in the DB, it would be `self.shortintroarabic`, so that's what should be assigned here.

Reply:
fixed